omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java

```diff
@@ -52,38 +52,38 @@
 @Resource(name = RestConstants.VERSION_1 + "/moduleaction", supportedClass = ModuleAction.class, supportedOpenmrsVersions = {
         "1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*" })
 public class ModuleActionResource1_8 extends BaseDelegatingResource<ModuleAction> implements Creatable {
-	
+
 	/**
 	 * ModuleFactoryWrapper is used for testing purposes.
 	 */
 	private ModuleFactoryWrapper moduleFactoryWrapper = new ModuleFactoryWrapper();
-	
+
 	public void setModuleFactoryWrapper(ModuleFactoryWrapper moduleFactoryWrapper) {
 		this.moduleFactoryWrapper = moduleFactoryWrapper;
 	}
-	
+
 	/**
 	 * Overriding create directly, because ModuleFactory requires ServletContext to execute any
 	 * action
 	 */
 	@Override
 	public Object create(SimpleObject post, RequestContext context) throws ResponseException {
 		moduleFactoryWrapper.checkPrivilege();
-		
```

Comment:
Can you remove all these formatting changes?

Reply:
My editor automatically removes white space that has been left in which ever file I modify. Re-editing the file will bring the same results.

Reply:
Can you turn that off?

Reply:
Okay

Reply:
Have failed to shut it off. The setting that I thought does that is failing to go off.

Reply:
Did you ask your teammates for help?

Reply:
I've fixed the formatting issue